hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -198,6 +199,9 @@ protected void startProcess(ExecutorDriver driver, Task task) {
     if (task.getProcess() == null) {
       try {
         ProcessBuilder processBuilder = new ProcessBuilder("sh", "-c", task.getCmd());
```

Comment:
Instead of overriding the HADOOP_HEAPSIZE here, we should set it properly when we create the Executor, https://github.com/mesosphere/hdfs/blob/master/hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java#L143. Our environment variables should be set in one place.